For `actionable` cb plugin also hide ok for itemized results

towolf opened the pull request:
##### Issue Type:
- Bugfix Pull Request
##### Ansible Version:

```
ansible 2.1.0 (hide_ok_items_in_actionable e51c234dde) last updated 2016/03/15 11:27:35 (GMT +200)
  lib/ansible/modules/core: (devel eef9c54874) last updated 2016/03/14 11:36:44 (GMT +200)
  lib/ansible/modules/extras: (devel be66e9d297) last updated 2016/03/14 11:36:45 (GMT +200)
  config file = /etc/ansible/ansible.cfg
  configured module search path = Default w/o overrides
```
##### Summary:

The purpose of the `actionable` callback plugin is to hide uninteresting
results.

Also hide the ok results when the task was itemized.
##### Example output:

```
TASK [template itemized] *******************************************************
changed: [localhost] => (item=[u'baz', u'quux'])
--- before
+++ after: dynamically generated
@@ -0,0 +1 @@
+bar


PLAY RECAP *********************************************************************
localhost                  : ok=4    changed=1    unreachable=0    failed=0  
```

jpic commented on lib/ansible/plugins/callback/actionable.py:
```diff
@@ -64,8 +64,11 @@ def v2_playbook_on_include(self, included_file):
+            self.display_task_banner()
+            self.super_ref.v2_playbook_item_on_ok(result)
+        else:
+            pass
```
Could you remove `else: pass` ?

towolf commented:
Updated PR to remove extraneous pass condition. Also removed it at the location above where I got it from.

jpic commented:
LGTM

towolf committed:
For `actionable` cb plugin also hide ok for itemized results

The purpose of the `actionable` callback plugin is to hide uninteresting
results.

Also hide the ok results when the task was itemized.

towolf commented:
Updated PR, because @jimi-c made a commit that conflicted with mine.
